privatebin/privatebin.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM janitortechnology/ubuntu-dev
+MAINTAINER PrivateBin <support@privatebin.org>
+
+# Download deps
+RUN sudo apt-get update && \
+    sudo apt-get -y install --no-install-recommends nginx php-fpm php-gd php-sqlite3 phpunit && \
+    npm install -g mocha jsverify jsdom@9 jsdom-global@2 mime-types && \
+
+# Configure php-fpm
+    sudo mkdir -p /run/php && \
+    sudo sed -ri "s/^;daemonize = .*$/daemonize = no/" /etc/php/7.0/fpm/php-fpm.conf && \
+    sudo sed -ri "s/^(user|group) = .*$/\1 = user/" /etc/php/7.0/fpm/pool.d/www.conf && \
+
+# Configure supervisord
+    sudo sh -c "echo '[program:php-fpm]\n\
+user = user\n\
+command = sudo /usr/sbin/php-fpm7.0\n\
+autorestart = true\n\
+[program:nginx]\n\
+user = user\n\
+command = sudo /usr/sbin/nginx -g \"daemon off;\"\n\
+autorestart = true' >> /etc/supervisord.conf" && \
+
+# Prepare Git repository & clean up
+    git clone https://github.com/PrivateBin/PrivateBin /home/user/privatebin && \
```

Comment:
Nit: I feel like this `git clone` could be run in a separate `RUN` command (you don't have to do *all* the setup in a single `RUN` command). This would make this file a bit more readable.

In general it's ok to use as many `RUN` instructions as you want (except for deleting files, this needs to be in the same `RUN` where they're created, otherwise they'll be captured in a Docker image layer).